docs/Getting_Involved/Newcomers.rst

```diff
@@ -0,0 +1,116 @@
+Welcome to the Newcomers guide!
+===============================
+
+This guide will help you to get involved and teach you how to make your first
+contribution to coala!
+
+Meet the community!
+-------------------
+
+To get started, the first way is to meet the community. We use gitter to
+communicate, and there the helpful community will guide you.
+
+::
+
+    https://gitter.im/coala-analyzer/coala
+
+
+Congratulations! You are now part of our community.
+
+Start working
+-------------
+
+Let us know that you are willing to contribute and you will be added to our
+organization on github. This is your first step towards contributing.
+The invitation will be sent upon mail and you will have to accept it to join.
+
+Now that you are part of our organization, you can start working on issues.
+
+::
+
+    https://github.com/coala-analyzer
+
+
+Get help with git
+-----------------
+
+We use github to manage our repository. If you're not familiar with git, we
+strongly recommend following a tutorial, such as:
+
+::
+
+    https://try.github.io/levels/1/challenges/1
+
+
+If there's anything unclear, or you are encountering problems, feel free
+to contact us on git, and we will help you!
+
+Start working
+-------------
+
+Now is the time to pick an issue.
+
+::
+
+    https://github.com/coala-analyzer/coala/issues
+
+
+You can also filter on git to make it show only newcomer issues.
+Here are two links that will lead you to only newcomers issues:
+
+::
+
+    For coala project: https://github.com/coala-analyzer/coala/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer
+
+    Also, for our coala-bears: https://github.com/coala-analyzer/coala-bears/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer
+
+    For more information about what bears are, please check the following link:
+    http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html
+
+The easy issues that will help you get started are labeled as
+"difficulty/newcomer" and are only there to give you a glimpse of how it is
+working with us. Now pick an issue, and if you want to fix it, then leave
+a comment that you would like to get assigned. This way we don't have more
+people working on the same issue at the same time.
+Now you can start working on it.
+For more info on how to work correctly with git:
+
+::
+
+    https://github.com/coala-analyzer/coala/wiki/Getting-Started-to-Contribute#a-little-git-advice
+
+
+.. note::
+
+    Careful! For more information about creating commit messages, check:
```

Comment:
that sounds like a warning or something dangerous, can we make it a bit friendlier?